docs: fix the issue of incorrect links to referenced documents in the project

hansedong opened the pull request:
Fix issue: the main issue addressed was the incorrect document links referenced throughout the project.

faisalburhanudin commented:
hi @hansedong, maybe you can include changes for all markdown links ?

such as
- quick-start.MD
- backups.MD

hansedong commented:
@faisalburhanudin 
In the project, I have also found and fixed the issue of incorrect document links referenced elsewhere, please kindly review again.
However, what I need to point out is that there is a reference to `/docs/backups.md` in the `README.md` document. Although I changed `backups.MD` to `backups.md`, I did not find any files related to backups.md in the project. What should this place be changed to?

Haleygo commented on README.md:
```diff
@@ -18,18 +18,18 @@ So you can use familiar CRD objects: `ServiceMonitor`, `PodMonitor`, `Prometheus
  Operator simplifies VictoriaMetrics cluster installation, upgrading and managing.
 
- It has integration with VictoriaMetrics `vmbackupmanager` - advanced tools for making backups. Check backup [docs](/docs/backups.MD)
+ It has integration with VictoriaMetrics `vmbackupmanager` - advanced tools for making backups. Check backup [docs](/docs/backups.md)
```
```suggestion
 It has integration with VictoriaMetrics [vmbackupmanager](https://docs.victoriametrics.com/vmbackupmanager.html) - advanced tools for making backups. Check [Backup automation for VMSingle](./resources/vmsingle.md#backup-automation) or [Backup automation for VMCluster](./resources/vmcluster.md#backup-automation).```

Haleygo reviewed (changes requested):
Thanks for the fixes, plz see the suggestion.

hansedong commented:
@Haleygo Updated, thanks for the suggestion.

Haleygo reviewed (approved):
LGTM!
